src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -DNDEBUG")
```

Comment:
I think that the default build type is `-DCMAKE_BUILD_TYPE=Debug`.
If you want to turn on all the optimization flags, the programmer building the package should use `-DCMAKE_BUILD_FLAG=Release`.

I am not keen in changing the default behavior of `cmake`, but I am happy to do so if it turns out that we cannot build sleef with these two standard flavors.

@Naoki will be able to help out with the compiler flags once we get in trouble. For know leave the standard flags of `cmake`. It might be worth setting travis-ci to test the `Release` build, not the `Debug` one.